packages/gated-content/src/environments.ts

```diff
@@ -3,18 +3,21 @@ import { EthereumAddress } from '@lens-protocol/shared-kernel';
 import { SupportedChains } from './conditions/types';
 
 export type EnvironmentConfig = {
+  name: 'production' | 'development';
```

Comment:
Also we can consume the exported enum here aswell

Reply:
that can cause unwanted dependencies between packages, I don't know now, but I think it's not a good time to make changes like that just before releasing v1, let's create a follow-up ticket to consider these comments, wdyt?

Reply:
`EnvironmentConfig` is an internal anywhere else and this whole package `@lens-protocol/gated-content` is an internal (not meant to be used directly).

This simply because the Lens API is not open source at the moment. So nobody can produce a complete environment deployment of the protocol+api+custom modules+gated contracts. So making it configurable seems not necessary at this stage.

Environment config (here but also in other parts of the SDK) are groups of cohesive configuration that needs to account for all these aspect of they won't simply work. So at this point in time the public environment we are going to offer to the public are just 2: `production` and `development`.